instant.ts

```diff
@@ -66,9 +66,7 @@ async function runBashScript(path: string, filename: string, args: string[]) {
     }
     await promise
   } catch (err) {
-    console.error(`Error: Script ${filename} returned an error`)
-    console.log(err.stdout)
-    console.log(err.stderr)
+    console.error(`Script ${path}${filename} returned an error`)
```

Comment:
@tumbledwyer I know you wanted to prepend this with something like `[Error]`. It doesn't need to be done here since the go cli will prepend that to the error message...